src/composables/accounts.ts

```diff
@@ -39,7 +40,7 @@ const mnemonic = useStorageRef<string>(
   STORAGE_KEYS.mnemonic,
   {
     backgroundSync: true,
-    migrations: RUNNING_IN_TESTS ? [
+    migrations: RUNNING_IN_TESTS || IS_EXTENSION ? [
```

Comment:
It's OK, but what do you think about making the 2 arrays as 1 (less quirky way)? I mean instead choosing between 2 arrays that has some duplicated items just:

```
    migrations: [
      migrateMnemonicVuexToComposable,
      (!RUNNING_IN_TESTS && !IS_EXTENSION) && migrateMnemonicCordovaToIonic,
    ].filter(excludeFalsy),
```